Feisty/GameView/GameViewComponents/GameTableViewCell.swift

```diff
@@ -14,14 +14,9 @@ public class GameTableViewCell: UITableViewCell {
   @IBOutlet weak var cellView: UIView!
   @IBOutlet weak var txtViewGameName: UILabel!
   @IBOutlet public weak var txtGamePrice: UILabel!
+  @IBOutlet weak var gameImage: UIImageView!
   public var cellTappedAction: (() -> Void)?
 
-  private lazy var backGroundView: UIView = {
-    let view = UIView()
-    view.backgroundColor = .darkGray
-    return view
-  }()
-
```

Comment:
Dead code that set the background colour of the cell. No longer needed as the background colour of the cell is now set to clear.